### PR TITLE
chore(comms):  Bump ws_logaccess to latest version

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/comms/src/services/wsdl/ws_logaccess/v1.02/ws_logaccess.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    AND = 1,
+    OR = 2
+}
+
+export enum LogSelectColumnMode {
+    MIN = 0,
+    DEFAULT = 1,
+    ALL = 2,
+    CUSTOM = 3
+}
```
Comment (GordonSmith) [inline packages/comms/src/services/wsdl/ws_logaccess/v1.02/ws_logaccess.ts:30]: @jeclrsg should enums (and the types) be generated within the namespace?

File: packages/comms/src/services/wsLogaccess.ts [hunk 2]
```diff
@@ -1,8 +1,150 @@
...
+        return super.GetLogs(request);
+    }
+
+    GetLogsEx(request: GetLogsExRequest): Promise<GetLogsExResponse> {
+        const getLogsRequest: WsLogaccess.GetLogsRequest = {
+            Filter: {
+                leftBinaryFilter: {
+                    BinaryLogFilter: [{
+                        leftFilter: {
+                            LogCategory: LogAccessType.All,
```
Comment (jeclrsg) [inline packages/comms/src/services/wsLogaccess.ts:86]: Should this be checking for the presence of a category in the request and defaulting to the ALL if not found?

**Review by jeclrsg — COMMENTED** [review]
@GordonSmith Had a question about the LogCategory in GetLogsEx

File: packages/comms/src/services/wsdl/ws_logaccess/v1.02/ws_logaccess.ts (hunk 1 above)
Comment (GordonSmith) [reply packages/comms/src/services/wsdl/ws_logaccess/v1.02/ws_logaccess.ts:30]: @jeclrsg can you open a ticket for that and fix, thx.

**Review by jeclrsg — APPROVED** [review]

**Pull request merged**